cdap-distributions/src/COPYRIGHT/angular-bootstrap@1.1.2/COPYRIGHT

```diff
@@ -0,0 +1 @@
+The MIT License (MIT)
```

Comment:
Is this intended to be a single-line file?

Reply:
There is no license info on the repo for this dependency, I assumed based on other angular based dependencies that this is also MIT licensed. see [this link](https://github.com/angular-ui/bootstrap-bower)